---
title: Search Data
date_tdt: 2018-09-10T16:33:34.412Z
author: ''
layout: support_detail
permalink: /support/search-data/
published: true
isPublic_b: true
bannerSentence_txt: Support
pageType_txt: support

---



## Search Data

### File Repository & Filter/Facets

The File Repository is the primary method of accessing data in the Kids First Data Resource Portal. It provides an overview of data available in Kids First and offers users a variety of filters for identifying and browsing participants and files of interest. Users can access the File Repository section from the Kids First Data Portal top menu bars.

On the left, a panel of data facets allows users to filter participants and files using a variety of criteria. If facet filters are applied, the table on the right will display information about matching participants and files. If no filters are applied, the table on the right will display information about all available data.  

When the user applies filters, a banner appears above the table displaying the active filters that have been applied and provides access to share and save the query for later reference. 


### Advanced Filter Search 

The File Repository section provides access to additional data filters beyond the defaults. Filters corresponding to additional properties listed in the Kids First Data Dictionary can be added using the ALL FILTERS button available at the top of the filter panel. 

[IMAGE__images/938e28d0-b518-11e8-ba53-9bcf777eccbe.md]

The button opens a search window that allows the user to find an additional filter by name or value. Not all filters have values available for filtering; checking the “Show only fields with values” checkbox will limit the search results to only those that do.

### File Browser Column Definitions
Below are the definitions and descriptions of the column headers in the Kids First Data Resource Portal File Browser. Some of these overlap with definitions in the Data Dictionary and are noted as such where applicable.

The portal displays a list of default columns, and more can be added to the view by selecting the &ldquo;Columns&rdquo; drop down within the file browser. 

<table>
	<tbody>
		<tr>
			<td>
				<p><strong> Column </strong></p>
			</td>
			<td>
				<p><strong> Description </strong></p>
			</td>
			<td>
				<p><strong> Notes </strong></p>
			</td>
		</tr>
		<tr>
			<td>
				<p><strong> File ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The Kids First DRC unique identifier for the file. </span></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> These always start with GF_ for &ldquo;Genomic File&rdquo; </span></p>
			</td>
		</tr>
		<tr>
			<td>
				<p><strong> Participants ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The Kids First DRC unique identifier for the participant. </span></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> These always start with PT_ for &ldquo;Participant&rdquo; </span></p>
			</td>
		</tr>
		<tr>
			<td>
				<p><strong> Study Name </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> Proband </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> Family ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The Kids First DRC unique identifier for the family. </span></p>
			</td>
			<td>
				<ul>
					<li style="font-weight: 400;"><span style="font-weight: 400;"> These always start with FM_ for &ldquo;Family.&rdquo; </span></li>
					<li style="font-weight: 400;"><span style="font-weight: 400;"> All members of a family will have the same family ID. </span></li>
				</ul>
			</td>
		</tr>
		<tr>
			<td>
				<p><strong> Data Type </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;">See definition in Default File Filters <a href="#fileD">here.</a></span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> File Format </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;">See definition in Default File Filters <a href="#fileD">here.</a></span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> File Size </strong></p>
			</td>
			<td>
        <p><span style="font-weight: 400;"> See definition in File Filters <a href="#files">here.</a></span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> File Download (No label - always last column) </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> If you have access to download a file, you will see a clickable down pointing arrow button. If you do not have access to download a file, you will see a lock icon indicating you do not have access. </span></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> Clicking on the down pointing arrow will download the file. </span></p>
			</td>
		</tr>
		<tr>
			<td>
				<p><strong> Participant External ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The external ID of the participant provided by the investigator of the original study. This is a deidentified ID unique only within its given study. </span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> File Name </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The name of the file. </span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> File External ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The external ID of the file in the original study. This is often the file name. </span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> Aliquot External ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The external ID of the aliquot provided by the investigator of the original study. This is a deidentified ID unique only within its given study. </span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> Sample External ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The external ID of the sample/biospecimen provided by the investigator of the original study. This is a deidentified ID unique only within its given study. </span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> Biospecimen ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The Kids First DRC unique identifier for the biospecimen. </span></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> These always start with BS_ for &ldquo;Biospecimen&rdquo; </span></p>
			</td>
		</tr>
		<tr>
			<td>
				<p><strong> Tissue Type (Source Text) </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> Diagnosis (Source Text) </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>
				<p><strong> Study ID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The Kids First DRC unique identifier for the study. </span></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> These always start with SD_ for &ldquo;Study&rdquo; </span></p>
			</td>
		</tr>
		<tr>
			<td>
				<p><strong> Latest DID </strong></p>
			</td>
			<td>
				<p><span style="font-weight: 400;"> The Gen3 Document ID (DID) of the file. </span></p>
			</td>
			<td>&nbsp;</td>
		</tr>
	</tbody>
</table>

### Search Facet Definitions

<p><span style="font-weight: 400;">Below are the definitions and some general notes for the Kids First DRC Data Dictionary and search facets. As some of these definitions are common across many online genomic data resources, we have modeled some of our definitions after the Genomic Data Commons located </span><a href="https://docs.gdc.cancer.gov/Data_Dictionary"><span style="font-weight: 400;">here</span></a><span style="font-weight: 400;">. Any definition noted with a [1] has come from GDC.</span></p>

### Default Facets
<p><span style="font-weight: 400;">The portal provides two tabs of default facets to search over. The search terms have been divided into the categories of Clinical and File to help make searching easier for the user. The definitions below </span></p>
<a name="clinicalD"></a>
### Clinical Filters 
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
  <p><strong>Study Name</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The short name of the study derived from the Study Long Name.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Assigning short names for studies allows for easier searching on the portal.</span></p>
</td>
</tr>
<tr>
<td>
  <p><strong>Diagnosis Category</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">An overarching classification based on the Diagnosis Source Text to aide in quick searching over Cancer and Structural Birth Defects.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
  <p><strong>Diagnosis (Source Text)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Analysis, and recognition of the presence and nature of disease, condition, or injury from expressed signs and symptoms; also, the scientific determination of any kind; the concise results of such a study/investigation. [1]</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Data as obtained directly from the original study/investigation.</span></p>
</td>
</tr>
<tr>
<td>
  <p><strong>Family Composition</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">A calculated value based on the family members present with genomic data within a participant&rsquo;s pedigree. </span></p>
</td>
<td>
<ul>
<li style="font-weight: 400;"><span style="font-weight: 400;">Proband Only: Only the proband was sequenced:</span></li>
<li style="font-weight: 400;"><span style="font-weight: 400;">Duo: The proband plus one parent was sequenced.</span></li>
<li style="font-weight: 400;"><span style="font-weight: 400;">Duo+: The proband plus one parent plus other family members were sequenced (The two other members are not both biological parents. Ex: A proband, mother, and sister.)</span></li>
<li style="font-weight: 400;"><span style="font-weight: 400;">Trio: The proband, their mother, and their father were sequenced.</span></li>
<li style="font-weight: 400;"><span style="font-weight: 400;">Trio+: The proband, their mother, their father, and other family members were sequenced. (Ex: A proband, a mother, a father, and a sister)</span></li>
</ul>
</td>
</tr>
<tr>
<td>
  <p><strong>Proband</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The participant serving as the starting point for enrollment into study, often the first family member seeking medical attention. </span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
  <p><strong>Gender</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Text designations that identify gender. Gender is described as the assemblage of properties that distinguish people on the basis of their societal roles. This value is self-reported and may come from a form, questionnaire, interview, etc. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Race</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">An arbitrary classification of a taxonomic group that is a division of a species. It is characterized by shared hereditary, physical attributes and behavior, and in the case of humans, by common history, nationality, or geographic distribution. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
  <p><strong>Tissue Type (Source Text)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Text term directly from the original investigation that represents a description of the kind of tissue collected with respect to disease status or proximity to tumor tissue. [1]</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Data as obtained directly from the original study/investigation.</span></p>
</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<a name="fileD"></a>
### File Filters

<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
  <p><strong>Experiment Strategy</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The type of sequencing experiment performed on the biospecimen. </span></p>
</td>
<td>
<ul>
<li style="font-weight: 400;"><span style="font-weight: 400;">Whole Genome Sequencing (WGS)</span></li>
<li style="font-weight: 400;"><span style="font-weight: 400;">RNA Sequencing (RNA-Seq)</span></li>
<li style="font-weight: 400;"><span style="font-weight: 400;">Micro RNA Sequencing (miRNA-Seq)</span></li>
</ul>
</td>
</tr>
<tr>
<td>
  <p><strong>Harmonized Data</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Indicate if the file has been harmonized to the Kids First DRC standards so that it can be used alongside any other file in the DRC from any other study and sequencing center.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Kids First DRC harmonization pipelines currently align to GRCh38. Read more about harmonization here.</span></p>
</td>
</tr>
<tr>
<td>
  <p><strong>Data Type</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The high level type of data contained within the file.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
  <p><strong>File Format</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The technical specification of the file, typically the file extension of the file.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Family Shared Data Types</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">All members in the family share this data type.</span></p>
</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
### All Filters
<a name="files"></a>
### File
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><strong>ACL</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The access control list value assigned to the file based on data access committee (DAC) authorization. This obtained from dbGaP for NIH based datasets. &nbsp;</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Availability</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Value assigned that indicates whether or not the file is available for download within the DRC. This value does not take into account individual user&rsquo;s permissions.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, all data on the portal are currently available for download so this field is not populated. </span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Access</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The type of authorization required in order to download the file. Open Access files are available to any registered portal user. Controlled access files require approval &amp; authorization by dbGaP or the controlling DAC.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For more information on Data Access, please see the Support page here.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Created At</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The date that the file was created in the DRC&rsquo;s system.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently not populated with data.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Data Type</strong></p>
</td>
<td>
  <p><span style="font-weight: 400;">See definition in Default File Facets <a href="#fileD">here.</a></span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>File Format</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default File Facets</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>File Name</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The exact name of the file.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Harmonized Data</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default File Facets <a href="#fileD">here.</a></span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Modified At</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The date that the file was last modified in the DRC&rsquo;s system.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently not populated with data.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Reference Genome</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The reference genome by which the sequencing experiment was run against.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Unharmonized files from the various sequencing centers use various different reference genomes depending on when and where they were sequenced. The DRC harmonized files are aligned to GRCh38.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>File Size</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The measure of the size of the file in KB, MB or GB.</span></p>
</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
### Participant
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><strong>Alias Group</strong></p>
</td>
<td>&nbsp;</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently not populated with data.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Available Data Types</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The File data types available for the participants. </span></p>
</td>
<td>
<p><span style="font-weight: 400;">See Data Types definition under Files for a more detailed Data Type definition.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Consent Type</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The informed consent type that the participant agreed to at the time of sample collection. This is used to inform any data use limitations.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Ethnicity</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">An individual&rsquo;s self-described social and cultural grouping, specifically whether an individual describes themselves as Hispanic or Latino. </span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Gender</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Proband</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Race</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p><br /><br /></p>
### Biospecimen
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><strong>Age at Event (Days)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The age of the participant in days that the biospecimen was collected.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Analyte Type</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Text term that represents the kind of molecular specimen analyte. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Anatomical Site (Source Text)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The source text from the investigator that describes the disease site of the submitted sample. [1]</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Data as obtained directly from the original study/investigation. </span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Composition</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Text term that represents the cellular composition of the sample. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Concentration (mg/ML)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Numeric value that represents the concentration of analyte or aliquot extracted from the sample or sample portion, measured in milligrams per milliliter. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>NCIt ID Tissue Type</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The <a href="https://ncit.nci.nih.gov/ncitbrowser/">National Cancer Institute Thesaurus (NCIt)</a> ID associated with the Tissue Type (Source Text) value. Derived by matching the Tissue Type Source Text with NCIt lookups.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>NCIt ID Anatomical Site</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The <a href="https://ncit.nci.nih.gov/ncitbrowser/">National Cancer Institute Thesaurus (NCIt)</a> ID associated with the Anatomical Site (Source Text) value. Derived by matching the Anatomical Site Source Text value with NCIt lookups.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Participant&rsquo;s Biospecimens Dbgap Consent Code</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The dbGaP-assigned consent code used for access granting that is derived directly from the participant&rsquo;s consent.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">See Consent Type under participant for further definitions on consent.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Shipment Date</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Date the biospecimen was shipped to the sequencing center.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently not populated with data.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Shipment Origin</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Location/institution from where the biospecimen was shipped.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently not populated with data.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Spatial Descriptor</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Term to indicate precise, relative anatomical position from where the biospecimen was obtained.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently not populated with data.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Tissue Type (Source Text)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Text term directly from the original investigation that represents a description of the kind of tissue collected with respect to disease status or proximity to tumor tissue. [1]</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Data as obtained directly from the original study/investigation.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Uberon ID Anatomical Site</strong></p>
</td>
<td>
  <p><span style="font-weight: 400;">The <a href="http://uberon.github.io/about.html">Uberon ID</a> associated with the Anatomical Site (Source Text) value. Derived by matching the Anatomical Site Source Text value with Uberon lookups.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Volume (uL)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The volume in microliters of the analytes derived from the analyte(s) shipped for sequencing and characterization. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p><br /><br /></p>
### Diagnosis
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><strong>Age at Event (Days)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The participant&rsquo;s age in days that they were diagnosed with the disease.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Diagnosis</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">A calculated rollup of a participant&rsquo;s diagnoses. If the participant has only Cancer diagnoses, the participant&rsquo;s value is Cancer. If the participant has only Structural Birth Defect diagnoses, the value is Structural Birth Defects. </span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Diagnosis (Source Text)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
</td>
<td>
</td>
</tr>
<tr>
<td>
<p><strong>Diagnosis Category</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>ICD ID Diagnosis</strong></p>
</td>
<td>
  <p><span style="font-weight: 400;"><a href="http://www.who.int/classifications/icd/en/">ICD10</a> code for the diagnosis. </span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently not populated with data.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Mondo ID Diagnosis</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The <a href="http://obofoundry.org/ontology/mondo.html">Mondo ID</a> associated with the Diagnosis (Source Text) value. Derived by matching the Diagnosis Source Text value with Mondo ID lookups.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>NCIt ID Diagnosis</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The <a href="https://ncit.nci.nih.gov/ncitbrowser/">National Cancer Institute Thesaurus (NCIt)</a> ID associated with the Diagnosis (Source Text) value. Derived by matching the Diagnosis Source Text value with NCIt lookups.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Spatial Descriptor</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Term to indicate precise, relative anatomical position of the diagnosis.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently not populated with data.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Tumor Location (Source Text)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Text term from the investigator that describes the anatomic site of the tumor. [1]</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Data as obtained directly from the original study/investigation.</span></p>
</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
### Family
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><span style="font-weight: 400;">Family Composition</span></p>
</td>
<td>
  <p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
</td>
<td>
</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p><span style="font-weight: 400;">All other Family facets are a derivative of definitions above. The Family facets are files that belong to all members within a family. </span></p>
<p>&nbsp;</p>
### Outcome
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><strong>Age at Event (Days)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Participant&rsquo;s age in days of the Outcome event.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Disease Related</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Text value describing whether or not the participant&rsquo;s outcome is related to their disease. For example, whether their deceased status was due to their disease.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Vital Status</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The survival state of the participant.</span></p>
</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
### Phenotype
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><strong>Age at Event (Days)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Participant&rsquo;s age in days that the phenotype was observed.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Ancestral HPO IDs</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The <a href="https://hpo.jax.org/app/">Human Phenotype Ontology</a> value associated with the Participant Phenotype (Source Text) value. Derived by matching the Phenotype Source Text value with HPO lookups.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>External ID</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">External ID provided by the investigator of the original study for the Phenotype observation.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>HPO Phenotype Observed</strong></p>
</td>
<td>
  <p><span style="font-weight: 400;">Files for which the <a href="https://hpo.jax.org/app/">HPO ID</a> was positively observed.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Participant Phenotype (Source Text)</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The observable characteristics in a participant resulting from the expression of genes, environment factors, and their interactions. &nbsp;</span></p>
</td>
<td>
<p><span style="font-weight: 400;">Data as obtained directly from the original study/investigation. </span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Participants Phenotype HPO - HPO Phenotype Not Observed</strong></p>
</td>
<td>
  <p><span style="font-weight: 400;">Files for which the <a href="https://hpo.jax.org/app/">HPO ID</a> was negatively observed.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Participants Phenotype HPO - Snomed Phenotype Not Observed</strong></p>
</td>
<td>
  <p><span style="font-weight: 400;">Files for which the <a href="https://www.snomed.org/">Snomed</a> value associated with the Participant Phenotype (Source Text) was negatively observed. Derived by matching the Phenotype Source Text value with Snomed lookups.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Participants Phenotype HPO - Snomed Phenotype Observed</strong></p>
</td>
<td>
  <p><span style="font-weight: 400;">Files for which the <a href="https://www.snomed.org/">Snomed</a> value associated with the Participant Phenotype (Source Text) was positively observed. Derived by matching the Phenotype Source Text value with Snomed lookups.</span></p>
</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
### Study
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><strong>Data Access Authority</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The authoritative group responsible for providing access. </span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Release Status</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The status of the study within its Data Access Authority.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">For potential future use, currently populated with &ldquo;Pending&rdquo;.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Study Long Name</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The name of the study for which the original sample was sequenced and researched. Samples and participants are organized in the portal by their originating study.</span></p>
</td>
<td>
<p><span style="font-weight: 400;">IDs are the Kid First ID which starts with SD_*. If the study is a dbGaP study, the external ID for the study will be the PHS accession number.</span></p>
</td>
</tr>
<tr>
<td>
<p><strong>Study Name</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default Clinical Filters <a href="#clinicalD">here.</a></span></p>
</td>
<td>
</td>
</tr>
<tr>
<td>
<p><strong>Version</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The dbGaP version of the study.</span></p>
</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
### Sequencing Experiment
<table>
<tbody>
<tr>
<td>
<p><strong>Facet</strong></p>
</td>
<td>
<p><strong>Description</strong></p>
</td>
<td>
<p><strong>Notes</strong></p>
</td>
</tr>
<tr>
<td>
<p><strong>Experiment Date</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Date the sample was sequenced.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Experiment Strategy</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">See definition in Default File Filters <a href="#fileD">here.</a></span></p>
</td>
<td>
</td>
</tr>
<tr>
<td>
<p><strong>Instrument Model</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">The model of the sequencer used to obtain data.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Is Paired End</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Are there paired ends? [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Library Name</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Name of library. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Library Strand</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Library stranded-ness. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Max Insert Size</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Max number of bases found between paired-end adapters.</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Mean Insert Size</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Mean number of bases found between paired-end adapters. </span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Mean Read Length</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Mean length of the sequenced fragments [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Platform</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Name of platform used to obtain data. [1]</span></p>
</td>
<td>&nbsp;</td>
</tr>
<tr>
<td>
<p><strong>Total Reads</strong></p>
</td>
<td>
<p><span style="font-weight: 400;">Total number of reads from the sequencing experiment.</span></p>
</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
